tests/utils/test_install.py

```diff
@@ -0,0 +1,100 @@
+import mock
+import sys
+import types
+
+from tests.subprocesstest import SubprocessTestCase, run_in_subprocess
+
+from ddtrace.utils.install import (
```

Comment:
We should probably have this as `ddtrace.utils.import_hook` to be more clear, since `install` could be anything, and makes me think about `pip install` first before module hooks.